dotcom-rendering/src/layouts/ImmersiveLayout.tsx

```diff
@@ -312,8 +312,9 @@ export const ImmersiveLayout = (props: WebProps | AppProps) => {
 
 	const { absoluteServerTimes = false } = article.config.switches;
 	const inTagLinkTest =
-		isWeb && article.config.abTests.tagLinkDesignVariant === 'variant';
-
+		isWeb &&
+		article.config.abTests.tagLinkDesignVariant === 'variant' &&
+		article.tags.some((tag) => tag.id === 'football/euro-2024');
```

Comment:
I've switched to .some() here rather than .find() as we only need a boolean response, not a TagType